crates/cubecl-cuda/tests/constant_array.cu

```diff
@@ -0,0 +1,34 @@
+typedef unsigned int uint;
+
+extern "C" __global__ void kernel(float output_0[], uint info[]) {
+
+  int3 absoluteIdx = make_int3(blockIdx.x * blockDim.x + threadIdx.x,
+                               blockIdx.y * blockDim.y + threadIdx.y,
+                               blockIdx.z * blockDim.z + threadIdx.z);
+
+  uint idxGlobal =
+      (absoluteIdx.z * gridDim.x * blockDim.x * gridDim.y * blockDim.y) +
+      (absoluteIdx.y * gridDim.x * blockDim.x) + absoluteIdx.x;
+  uint rank = info[0];
+  uint rank_2 = rank * 2;
+  const float arrays_0[3] = {
+      uint(3),
+      uint(5),
+      uint(1),
+  };
```

Comment:
isn't strange that the values are casted to `uint`?

Reply:
I don't know what's up with that either, that's what the backend does when I just print the value. I might just force a cast to the output type.